docs/source/markdown/podman.1.md

```diff
@@ -75,110 +75,161 @@ If no identity file is provided and no user is given, podman defaults to the use
 Podman prompts for the login password on the remote server.
 
 Identity value resolution precedence:
- - command line value
- - environment variable `CONTAINER_SSHKEY`, if `CONTAINER_HOST` is found
- - `containers.conf`
-Remote connections use local containers.conf for default.
+ - command line value.
+ - environment variable `CONTAINER_SSHKEY`, if `CONTAINER_HOST` is found.
+ - `containers.conf`.
+Remote connections use local `containers.conf` for default.
 
-#### **--log-level**=*level*
+#### **--log-level**=***_error_** | debug | info | warn | fatal/panic*
 
-Log messages above specified level: debug, info, warn, error (default), fatal or panic (default: "error")
```

Comment:
This information is actually outdated. The default is now `warn`. Also the `trace` level is now supported which shows even more information than `debug`.

Reply:
Thanks! I forgot to verify that. I have updated it.